app/helpers/requests_helper.rb

```diff
@@ -5,12 +5,29 @@ module RequestsHelper
   ##
   # Generates the options needed to change a request's location
   def request_location_options(request)
-    options_for_select(
-      LibraryLocation.new(request.home_location)
-        .additional_pickup_libraries(request.pickup_library).index_by do |code|
-          Mylibrary::Application.config.library_map[code]
-        end
-    )
+    # TODO: after FOLIO launch remove this conditional wrapper
+    if request.is_a?(Folio::Request)
+      return options_for_select(
+        service_points(request),
+        # the second param here pre-selects the current service point in the dropdown
+        request.service_point_id
+      )
```

Comment:
for a function interface like this, I will sometimes mix positional and keyword params. so it would look like:
```rb
def options_for_select(service_points, selected: nil)
```
then when calling, the interface is a little clearer:
```rb
return options_for_select(service_points(request), selected: request.service_point_id)
```
i realize this PR doesn't have anything to do with that function, but it might be worth putting a `TODO: ` to refactor it later.

Reply:
Thanks @thatbudakguy ! Idk if you knew this was a built-in Rails method -- they define it as you suggest:
https://apidock.com/rails/v5.2.3/ActionView/Helpers/FormOptionsHelper/options_for_select

So I guess here I could add the keyword in to my call.

Reply:
I did not know that! If it's that easy and supports both APIs, then I guess go ahead and do the keyword version. Thanks!